test/WebSites/ActivatorWebSite/TagHelpers/HiddenTagHelper.cs

```diff
@@ -20,7 +20,7 @@ public class HiddenTagHelper : TagHelper
         {
             var content = await context.GetChildContentAsync();
 
-            output.Content = HtmlHelper.Hidden(Name, content).ToString();
+            output.Content.SetContent(HtmlHelper.Hidden(Name, content).ToString());
```

Comment:
nit: looks like there's a few more `ToString()` cases from here to the end of the file that should use `GetContent()`